staging/src/k8s.io/mount-utils/mount_windows.go

```diff
@@ -244,6 +244,11 @@ func (mounter *Mounter) IsLikelyNotMountPoint(file string) (bool, error) {
 	return true, nil
 }
 
+// CanSafelySkipMountPointCheck always returns false on Windows
+func (mounter *Mounter) CanSafelySkipMountPointCheck() bool {
+	return false
```

Comment:
I used `false` here to avoid changing any behavior on Windows; but looking at the `Unmount` implementation, I think this might be fine to return `true`?

Reply:
ya I see the same, it won't make a difference since windows `mounter.List` does nothing and has a todo  https://github.com/kubernetes/kubernetes/blob/9c151bb31a639663a1d79b4684b5e0a0a0488a2d/staging/src/k8s.io/mount-utils/mount_windows.go#L229 . i would leave it false, it is up to whoever eventually implements that `List` todo to figure out when/if CanSafelySkipMountPointCheck should ever return true